src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,84 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        try {
```

Comment:
Consider using try-with-resources for the `Files.deleteIfExists` operation as well. This will ensure that any file system resources are freed up immediately after the operation.

Reply:
remove this from public method